app/components/nypr-brick-card-row.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'ul',
+  classNames: 'brick_row brick_row--cards',
+  group: Ember.computed('items', function() {
```

Comment:
if you're not doing any kind of transformation, you can just pass in `items` under the name `group` like this `{{nypr-brick-card-row group=items}}`